src/types/modules/irc-framework/irc-framework.d.ts

```diff
@@ -0,0 +1,367 @@
+/* eslint-disable no-use-before-define */
```

Comment:
Can't wait to see this PR'd! 🙃 https://github.com/kiwiirc/irc-framework/issues/166#issuecomment-447922799